test/standalone/test-debuglet.js

```diff
@@ -195,11 +205,88 @@ describe(__filename, function(){
     debuglet.start();
   });
 
-  it('should de-activate when the server responds with isDisabled');
+  it('should de-activate when the server responds with isDisabled', function(done) {
```

Comment:
The test doesn't do precisely what this description specifies. It verifies that we are able re-register after a disabling by the server, but it doesn't verify that the debuglet went into a 'de-activate' state.